tests/unit/specs/plugins/vue-bem/testingEntitys.ts

```diff
@@ -8,7 +8,17 @@ export const delimiters = {
   modVal: '~',
 };
 
-export const blocks = {
+type Block = {
+  block: string;
+  mods?: Record<string, boolean | string | number>;
+  mixin?: string;
+}
+
+interface Blocks {
```

Comment:
todo: please use `type` instead. You can still extend this by using `type Foo = Blocks & {}`. `interface` should only be used in libraries, because they are shared on a global scope.